apps/web/pages/v2/settings/my-account/profile.tsx

```diff
@@ -124,15 +124,26 @@ const ProfileView = () => {
     deleteMeMutation.mutate({ password, totpCode });
   };
 
-  const formMethods = useForm({
-    defaultValues: {
-      avatar: user?.avatar || "",
-      username: user?.username || "",
-      name: user?.name || "",
-      email: user?.email || "",
-      bio: user?.bio || "",
-    },
-  });
+  const formMethods = useForm<{
```

Comment:
Going forward, suspense mode might be a better approach to avoid unnecessary useEffect.